glib/src/log.rs

```diff
@@ -972,16 +972,20 @@ pub fn log_variant(log_domain: Option<&str>, log_level: LogLevel, fields: &crate
 #[cfg_attr(docsrs, doc(cfg(unix)))]
 #[doc(alias = "g_log_writer_supports_color")]
 #[inline]
-pub fn log_writer_supports_color<T: AsRawFd>(output_fd: T) -> bool {
-    unsafe { from_glib(ffi::g_log_writer_supports_color(output_fd.as_raw_fd())) }
+pub fn log_writer_supports_color(output_fd: impl AsFd) -> bool {
```

Comment:
Should keep unsafe versions of this and below around

Reply:
I don't think so, you can unsafely use the safe versions with BorrowedFd::borrow_raw or similar.

Reply:
Hm good point. But then we should probably remove all the others too?